File: Bimestre 2/arreglobidimensional/MatricesPython/menorfila.py
n = int(input("Ingrese el tamaño de la matriz cuadrada: "))
A = []
l=0
k=0
print("Ingresar los elementos de la matriz A")
for i in range(0, n):
    A.append([])
    for j in range(0, n):
        pos = int(input("A[" + str(i) + "]["+str(j)+"]= "))
        A[i].append(pos)
print('Matriz A')
for i in range(0, n):
    for j in range(0, n):
        print(A[i][j], end='   ')
    print('  ')

for i in range(0, n):
    menor = A[i][0]
    for j in range(0, n):
        if A[i][j]< menor:
            menor = A[i][j]
            k=i
            l=j

    print("El elemento menor de la fila " +str(i) + " es " +str(menor)+" y está en A["+str(i)+"]["+ str(l) +"]")
    k=0
    l=0

File: Bimestre 2/arreglobidimensional/Matrices/src/matrices/menorfila.java
 /*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package matrices;
import java.util.Scanner;
/**
 *
 * @author kevin
 */
public class menorfila {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        int n, j,i,k,l, menor;
        menor = 0; n=0; l=0; k=0;
        System.out.print("Ingrese el tamaño de la matriz cuadrada: ");
        n = leer.nextInt();
        
        int[][] A = new int [n][n];
        for(i=0;i<n;i++){
            for(j=0;j<n;j++){
                System.out.print("A ["+i+"] ["+j+"] = ");
                A[i][j] = leer.nextInt();
                }
            }
        
        
        for (i = 0; i < n; i++) {
            for (j = 0; j <n ; j++) {
                System.out.print(A[i][j] + "  ");
                
            }
            System.out.println("");
        }
        
        for(i=0;i<n;i++){
            menor = A[i][0];
            for(j=0;j<n;j++){
                if(A[i][j]< menor){
                    menor = A[i][j];
                    
                           
                }
            }
            System.out.println("El elemento menor de la fila " +i + " es "
                    +menor+" y está en A["+i+"]["+ l +"]");
            
            
            
        }
        
    }
}
